server/routerlicious/packages/protocol-base/src/protocol.ts

```diff
@@ -136,7 +177,48 @@ export class ProtocolOpHandler {
         return {
             sequenceNumber: this.sequenceNumber,
             minimumSequenceNumber: this.minimumSequenceNumber,
-            ...this.quorum.snapshot(),
+            ...this._quorum.snapshot(),
         };
     }
 }
+
+export class ProtocolOpHandlerWithClientValidation extends ProtocolOpHandler {
+    constructor(
```

Comment:
I think you should be able to remove constructor and it should just work?
I'd remove "public" keyword from arguments below - they are already public in base class.